### PR TITLE
MIL COMMON: clean redundant packages and prepare for great merge & sonar OS

### PR DESCRIPTION
**Pull request by kev-the-dev**
* remove all/part of sub8_vision_lib sub8_ros_tools uf_common and passive sonar packages
* make namespace changes to use mil_common versions of the above where possible

Why I think Nothings broken
* Project compiles running my branch of mil_common and this PR
* git grep for the following strings return nothing: ""sub8_tools" "sub8_ros_tools" "sub8_mil_tools" "uf_common"
* running ```sub8_missions run_common list``` shows all missions without import errors

Obviously don't merge until uf-mil/mil_common#15 is merged and uf-mil/mil_common#28

### CODE REVIEW

File: command/sub8_missions/tools/rviz_visualizer.py [hunk 1]
```diff
@@ -9,10 +9,10 @@
 
 from geometry_msgs.msg import Pose, Vector3
 from std_msgs.msg import ColorRGBA, Float64
-from uf_common.msg import Float64Stamped  # This needs to be deprecated
+from mil_msgs.msg import RangeStamped, DepthStamped
 #from sub8_alarm import AlarmListener, AlarmBroadcaster
 from ros_alarms import AlarmBroadcaster, AlarmListener
-import sub8_ros_tools as sub8_utils
+import mil_ros_tools as sub8_utils
```
Comment (DSsoto) [inline command/sub8_missions/tools/rviz_visualizer.py]: This should not be imported as sub8_utils.

File: command/sub8_missions/tools/waypoint_utility.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import sys
 import rospy
 import nav_msgs.msg as nav_msgs
-import sub8_ros_tools as sub8_utils
+import mil_ros_tools as sub8_utils
```
Comment (DSsoto) [inline command/sub8_missions/tools/waypoint_utility.py]: This should not be imported as sub8_utils.

File: command/sub8_missions/tools/waypoint_utility.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 from scipy import linalg
 
 import actionlib
-import uf_common.msg as uf_common_msgs
+import mil_msgs.msg as mil_msgs_msgs
```
Comment (DSsoto) [inline command/sub8_missions/tools/waypoint_utility.py]: It soesnt make sense to import as a name and that name is longer. How about:
>  from mil_msgs.msg import MoveToAction, MoveToGoal

File: drivers/magnetic_compensation/sub8_magnetic_hardsoft_compensation/scripts/generate_config [hunk 4]
```diff
@@ -10,7 +10,7 @@ import yaml
 import roslib
 import rosbag
 from tf import transformations
-import sub8_ros_tools as sub8_utils
+import mil_ros_tools as sub8_utils
```
Comment (DSsoto) [inline drivers/magnetic_compensation/sub8_magnetic_hardsoft_compensation/scripts/generate_config]: Ditto

File: simulation/sub8_simulation/sub8_sim_tools/physics/vehicle.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import rospy
-import sub8_ros_tools as sub8_utils
+import mil_ros_tools as sub8_utils
```
Comment (DSsoto) [inline simulation/sub8_simulation/sub8_sim_tools/physics/vehicle.py]: Ditto

**Review by DSsoto — CHANGES_REQUESTED** [review]
Requesting a couple of minor changes.

**Comment by sentree** [comment]
This built on the CI and passed all unit tests except the ros_alarms test that always fails.
-- Your friendly temporary CI guy

**Review by DSsoto — APPROVED** [review]

**Pull request merged**